src/renderers/shaders/ShaderLib.js

```diff
@@ -299,12 +299,15 @@ ShaderLib.physical = {
 	uniforms: mergeUniforms( [
 		ShaderLib.standard.uniforms,
 		{
-			transparency: { value: 0 },
 			clearcoat: { value: 0 },
+			clearcoatMap: { value: null },
 			clearcoatRoughness: { value: 0 },
-			sheen: { value: new Color( 0x000000 ) },
+			clearcoatRoughnessMap: { value: null },
+			clearcoatNormalMap: { value: null },
 			clearcoatNormalScale: { value: new Vector2( 1, 1 ) },
 			clearcoatNormalMap: { value: null },
```

Comment:
`google-closure-compiler` complains about `clearcoatNormalMap` being used twice, failing the build
```
build/three.js:15412: ERROR - [JSC_DUPLICATE_OBJECT_KEY] Object literal contains illegal duplicate key "clearcoatNormalMap", disallowed in strict mode
				clearcoatNormalMap: { value: null },
				^^^^^^^^^^^^^^^^^^

1 error(s), 0 warning(s)
```

Reply:
Fixed, thanks!